utils/vectorscan/Makefile

```
@@ -0,0 +1,79 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=vectorscan
+PKG_VERSION:=5.4.9
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/VectorCamp/vectorscan/tar.gz/$(PKG_NAME)/$(PKG_VERSION)?
+PKG_HASH:=e61c78f26a9d04ccffab0df1159885c4503fc501172402c57f7357a2126ea3c6
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_NAME)-$(PKG_VERSION)
+
+PKG_MAINTAINER:=John Audia <therealgraysky@proton.me>
+PKG_LICENSE:=Intel
+PKG_LICENSE_FILES:=LICENSE
+PKG_BUILD_PARALLEL:=1
+
+PKG_BUILD_DEPENDS:=ragel/host python3/host boost/host
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/cmake.mk
+
+define Package/vectorscan
+  CATEGORY:=Utilities
+  SECTION:=utils
+  TITLE:=Vectorscan
+  URL:=https://github.com/VectorCamp/vectorscan
+  DEPENDS:=@aarch64 @x86
```

```suggestion
  DEPENDS:=@(aarch64||arm||i386||i686||x86_64)
```
If you don't know, try to do some research and ask nicely. Don't just guess.

---

This conversation is marked as resolved by @graysky2 , but in https://github.com/openwrt/packages/pull/27382 , I see that this was not addressed. Vectorscan uses: ``DEPENDS_COMMON:=@(x86_64||aarch64)``